inst/tests/tests.Rraw

```diff
@@ -14473,8 +14473,8 @@ options(datatable.rbindlist.check=NULL)  # this option is set to NULL at the top
 if (.Platform$OS.type == 'windows') local({
   lc_collate <- Sys.getlocale(c('LC_COLLATE'))
   lc_ctype <- Sys.getlocale(c('LC_CTYPE'))
-  Sys.setlocale('LC_COLLATE', "Chinese (Simplified)_China.936")
-  Sys.setlocale('LC_CTYPE', "Chinese (Simplified)_China.936")
+  suppressWarnings(Sys.setlocale('LC_COLLATE', "Chinese (Simplified)_China.936"))
```

Comment:
For future reference, please add either a TODO to remove the suppression, or a comment about why suppressWarnings() is necessary here.